--- src/Makefile.am	2011-05-18 14:00:25.000000000 +0200
+++ src/Makefile.am.oden	2011-05-29 11:36:25.296391444 +0200
@@ -28,8 +28,8 @@
 AM_CPPFLAGS=-I$(top_srcdir)/include $(XML2_CFLAGS) $(SSL_CFLAGS) 
 libyaz_la_LIBADD = $(SSL_LIBS) $(TCPD_LIBS)
 libyaz_server_la_LIBADD = libyaz.la
-libyaz_icu_la_CPPFLAGS = $(AM_CPPFLAGS) $(ICU_CPPFLAGS) -I$(top_srcdir)/libstemmer_c/include
-libyaz_icu_la_LIBADD = libyaz.la $(ICU_LIBS)
+libyaz_icu_la_CPPFLAGS = $(AM_CPPFLAGS) $(ICU_CPPFLAGS) -I/usr/include/libstemmer
+libyaz_icu_la_LIBADD = libyaz.la $(ICU_LIBS) -lstemmer
 
 AM_YFLAGS=-p cql_
 
@@ -116,22 +116,9 @@
 
 libyaz_server_la_LDFLAGS=-version-info $(YAZ_VERSION_INFO)
 
-STEMMER_SOURCES = \
-	../libstemmer_c/include/libstemmer.h \
-	../libstemmer_c/libstemmer/libstemmer.c \
-	../libstemmer_c/libstemmer/modules.h \
-	../libstemmer_c/runtime/api.c \
-	../libstemmer_c/runtime/api.h \
-	../libstemmer_c/runtime/header.h \
-	../libstemmer_c/runtime/utilities.c \
-	../libstemmer_c/src_c/stem_UTF_8_porter.c \
-	../libstemmer_c/src_c/stem_UTF_8_porter.h \
-	../libstemmer_c/src_c/stem_UTF_8_english.c \
-	../libstemmer_c/src_c/stem_UTF_8_english.h
-
 libyaz_icu_la_SOURCES = icu_chain.c icu_utf16.c icu_utf8.c stemmer.c \
-	icu_transform.c icu_casemap.c icu_tokenizer.c icu_sortkey.c \
-	$(STEMMER_SOURCES)
+	icu_transform.c icu_casemap.c icu_tokenizer.c icu_sortkey.c
+
 libyaz_icu_la_LDFLAGS=-version-info $(YAZ_VERSION_INFO)
 
 # Rules for Z39.50 V3
